docs/data/joy/components/chip/ChipUsage.js

```diff
@@ -0,0 +1,341 @@
+                return setProps((latestProps) => ({
+                  ...latestProps,
+                  size: event.target.value,
+                }));
+              }}
+              sx={{ flexWrap: 'wrap', gap: 1 }}
+            >
+              {['sm', 'md', 'lg'].map((value) => {
+                const checked = size === value;
+                return (
+                  <Chip
+                    key={value}
+                    variant="plain"
+                    color="neutral"
+                    size="sm"
+                    sx={{ bgcolor: 'background.body' }}
+                  >
+                    <Radio
+                      size="sm"
+                      variant="outlined"
+                      color="neutral"
+                      label={
+                        <Typography
+                          endDecorator={checked ? <Check fontSize="md" /> : null}
+                        >
+                          {value}
+                        </Typography>
+                      }
+                      value={value}
+                      disableIcon
+                      overlay
+                    />
+                  </Chip>
+                );
+              })}
+            </RadioGroup>
+          </Box>
+          <Box>
+            <Typography
+              id="badge-disabled-selector"
+              level="body2"
+              fontWeight="lg"
+              mb={1}
+            >
+              disabled
+            </Typography>
+            <RadioGroup
+              row
+              name="badge-disabled"
+              aria-labelledby="badge-disabled-selector"
+              value={disabled}
+              onChange={(event) => {
+                return setProps((latestProps) => ({
+                  ...latestProps,
+                  disabled: event.target.value === 'true',
+                }));
+              }}
+              sx={{ flexWrap: 'wrap', gap: 1 }}
+            >
+              {[false, true].map((value) => {
+                const checked = disabled === value;
+                return (
+                  <Chip
+                    key={`${value}`}
+                    variant="plain"
+                    color="neutral"
+                    size="sm"
+                    sx={{ bgcolor: 'background.body' }}
+                  >
+                    <Radio
+                      size="sm"
+                      variant="outlined"
+                      color="neutral"
+                      label={
+                        <Typography
+                          endDecorator={checked ? <Check fontSize="md" /> : null}
+                        >
+                          {`${value}`}
+                        </Typography>
+                      }
+                      value={`${value}`}
+                      disableIcon
+                      overlay
+                    />
+                  </Chip>
+                );
+              })}
+            </RadioGroup>
+          </Box>
```

Comment:
Should we add a `TextField` for customizing the children?

Reply:
Sure. Done